src/main/java/pingmebot/task/Events.java

```diff
@@ -0,0 +1,68 @@
+package pingmebot.task;
+
+/**
+ * A category of task that has a task description, and a start and end date or time to complete the task by.
+ */
+public class Events extends Task {
+    protected String start;
+    protected String end;
+    protected String description;
+
+    /**
+     * Creates an object with the initialisation of the task's description, start and end date or time.
+     *
+     * @param description The task's description.
+     * @param start The starting date or time of the task.
+     * @param end The ending date or time of the task.
+     */
+    public Events(String description, String start, String end) {
+        super(description);
+        this.description = description;
+        this.start = start;
+        this.end = end;
+    }
+
+    /**
+     * Returns a string representation of the Events task.
+     *
+     * @return A string representation of the Events task.
+     */
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from:" + start + " to:" + end + ")";
```

Comment:
Could try to be more consistent with whether or not to add `this` in front of instance fields.